Added support for downloading effect and optional icon file

thammer opened the pull request.

mungewell commented:
I can see the positives of having a way in the console script to download some/all files, but I am not convinced the saving of files to local disk should be part of the `zoomzt2` class. Can you adjust so that the actual saving is under `main()` or maybe a standalone `def` instead? 

On up/downloading the extra files (ZIC and IIR), yes. And see you have checked whether they already exist - pedal's file system will allow uploading multiple files with same name and will consume increasing disk space.

Q. When we `--uninstall` do we also want to remove the extra files?

thammer commented:
Good points.

I have moved download_and_save_file() and download_and_save_all_files() to the global namespace, as you suggested.

I have modified the behavior for uninstall and uninstallonly so that it optionally deletes the ZIC and ZIR files. The options are the same as for install/installonly, --include-zic and --include-zir.

thammer commented:
I have tested install and uninstall of effect with .ZIC and .ZIR, examined file listing with zoomzt2-gyi.py before and after install and uninstall, and I have examined the effects that appear on the Library screen on the pedal itself. Everything seems to be working as expected.
